pkgs/tools/misc/microplane/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "microplane";
+  version = "0.0.25";
+
+  src = fetchFromGitHub {
+    owner = "Clever";
+    repo = "microplane";
+    rev = "v${version}";
+    sha256 = "0vynkv3d70q8d079kgdmwbavyyrjssqnd223dl1rikyy7sd5sli8";
+  };
+
+  goPackagePath = "github.com/Clever/microplane";
+
+  subPackages = ["."];
+
+  # Regenerate deps.nix with the following steps:
+  #
+  # git clone git@github.com:Clever/microplane.git
+  # cd microplane
+  # git checkout v<version>
+  # dep2nix
+
+  goDeps = ./deps.nix;
+
+  buildFlagsArray = ''
+    -ldflags=
+    -X main.version=v${version}
```

Comment:
```suggestion
    -ldflags="-s -w -X main.version=v${version}"
```

Reply:
Super fast :zap: thanks!

Just made the change.